emails/payin_directdebit_failed.spt

```diff
@@ -1,13 +1,13 @@
-{{ _("The direct debit payment to your Liberapay wallet has failed") }}
+{{ _("The direct debit payment to your Liberapay wallet failed") }}
 
 [---] text/html
 <p>{{ _(
-    "The attempt to transfer {0} from your bank account has failed with the following error message:",
+    "The attempt to transfer {0} from your bank account failed because:",
```

Comment:
Same as above

Reply:
No, you cannot use a colon just after "because". See the punctuation rules on colons.